test/Microsoft.AspNetCore.SignalR.Tests/Microsoft.AspNetCore.SignalR.Tests.csproj

```diff
@@ -1,15 +1,14 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netcoreapp2.0;net461</TargetFrameworks>
-    <TargetFrameworks Condition="'$(OS)' != 'Windows_NT'">netcoreapp2.0</TargetFrameworks>
+    <TargetFrameworks>$(StandardTestTfms)</TargetFrameworks>
 
     <!--
       Workaround for "Use executable flags in Microsoft.NET.Test.Sdk" (https://github.com/Microsoft/vstest/issues/792).
       Remove when fixed.
     -->
     <HasRuntimeOutput>true</HasRuntimeOutput>
-    <RuntimeIdentifier Condition="'$(TargetFramework)' != 'netcoreapp2.0'">win7-x64</RuntimeIdentifier>
```

Comment:
Running this for x64 produces an error that says it mismatches the test platform architecture.

Reply:
Is it a regression then? Because it used to work just fine...

Reply:
What

Reply:
https://github.com/aspnet/BuildTools/issues/471

Reply:
If it's super important to target the tests to run on x64, I can create a follow up work item to revert this once https://github.com/aspnet/BuildTools/issues/471 is resolved.

Reply:
@pranavkm yes lets file a bug to bring them back please.

Reply:
https://github.com/aspnet/SignalR/issues/1133

Reply:
I am not concerned about the tests too much as long as they run on full framework. The thing I am concerned about (not understanding the root cause) is whether customers will have the same issue with their projects targeting full framework.

Reply:
It's a change in how we run the tests in our SDK. It's not an issue with the actual binary.